README.md

```diff
@@ -1,18 +1,48 @@
+
+Build-task.yaml (https://github.com/kabanero-io/collections/blob/master/incubator/common/pipelines/default/build-task.yaml) builds a container image from the artifacts in the git-source repository using buildah.  Once the image is built, it will publish it to the docker-image URL using buildah.
+
+Deploy-task.yaml (https://github.com/kabanero-io/collections/blob/master/incubator/common/pipelines/default/build-task.yaml) will modify the app-deploy.yaml, which describes the deployment options for the application, to point to the image that was published and will deploy the application using the appsody operator.
+
+For additional tasks and more information about kabanero pipelines, visit the kabanero-pipelines (https://github.com/kabanero-io/kabanero-pipelines) repo.
+
+# Running the pipelines
+
+## Prereqs
 
 You have the Kabanero foundation installed on Red Hat Origin Community Distribution of Kubernetes (OKD) or OpenShift Container Platform (OCP) cluster.  It has the necessary Kabanero, Isito, Knative, and Tekton components installed.  Please refer to https://github.com/kabanero-io/kabanero-foundation for more details on installing the Kabanero foundation.
 
 Identify the Tekton Dashboard URL after your have completed the installation.  You can login to your OKD cluster and run ```oc get routes``` to find this or check in the OKD console.  This is useful for a few of the steps documented below.  You can find more details about the dashboard at https://github.com/tektoncd/dashboard.
 
-NOTE: Everything is assumed to be running in `kabanero` namespace.
+NOTE: By default the pipelines will run in `kabanero` namespace and deploy the application there as well.  If you would like to deploy the application in a different namespace, please update the app-deploy.yaml to point to the namespace to deploy to.
+
+### Setup dynamic volume provisioning or a persistent volume
 
-### Create a persistent volume
-The persistent volume is used by the pipelines.  An example pv definition is provided.  Update path and other values in pv.yaml to suit your requirements.
+Tekton pipelines requires a volume to be configured that will be used by the framework to share data across tasks.  The build task, which leverages buildah, also requires a volume mount.  The minimum required volume size is 5 Gi.  
+
+If your cluster is running in a public cloud, dynamic volume provisioning is the easier and preferred option.  Please refer to the following doc that describes how to point to the storage class using a config map.   https://github.com/tektoncd/pipeline/blob/master/docs/install.md#how-are-resources-shared-between-tasks
+
+Note:  If this is not configured and based on how your cloud provider's default storage class is configred, your pipelines might create a new volume for each run which will increase your pipeline run execution time.
+
```

Comment:
just wondering: how about adding a final step in those tasks that require a `pv`, to delete it?
[ rationale: while `pv` are meant to be persistent by design, here we precisely know that the `persistence` is scopped between the tasks / steps - so!]